rebar.config.script

```diff
@@ -1,8 +0,0 @@
-Relx0 = lists:keyfind(relx, 1, CONFIG),
```

Comment:
I see references to this file in:
* Dockerfile
* .dockerignore

Reply:
I removed these in https://github.com/aeternity/epoch/pull/1330 There I also started additional CI checks.

(BTW CI checks did not run for this PR - I tracked this as https://www.pivotaltracker.com/story/show/158938024 )

Reply:
K, I'll get rid of rebar.config.script from dockerfile and dockerignore.

Reply:
Oh, nevermind, just saw you already did that :). I'll get the docs up sometime this morning and comment in that PR.